manifests/init.pp

```diff
@@ -353,7 +353,7 @@
   Array $ssl_ciphers                                                                               = $rabbitmq::params::ssl_ciphers,
   Boolean $stomp_ensure                                                                            = $rabbitmq::params::stomp_ensure,
   Boolean $ldap_auth                                                                               = $rabbitmq::params::ldap_auth,
-  String $ldap_server                                                                              = $rabbitmq::params::ldap_server,
+  Variant[String,Array[String]] $ldap_server                                                                              = $rabbitmq::params::ldap_server,
```

Comment:
Please update it to this:
```suggestion
  Variant[String[1],Array[String[1]]] $ldap_server                                                             = $rabbitmq::params::ldap_server,
```

Also please ensure that the `=` are all aligned

Reply:
@skrussell Could you please apply the suggestion of @bastelfreak . After that we should be able to merge this pull request.